scripts/generate_architecture_drift_report.sh

```
@@ -109,9 +122,15 @@ hotspot_breach=0
 hotspot_count=0
 boundary_count=0
+tight_hotspots=()
+watch_hotspots=()
+mixed_class_hotspots=()
+tight_hotspot_summary="none"
+watch_hotspot_summary="none"
+mixed_class_hotspot_summary="none"
 hotspot_rows="$(architecture_hotspot_rows)" || exit 1
 
-while IFS='|' read -r key file lines max_lines line_status functions max_functions fn_status; do
+while IFS='|' read -r key file classes lines max_lines line_status functions max_functions fn_status peak_usage pressure; do
```

Shellcheck (0.11.0)

[warning] line_status appears unused. Verify use (or export if used externally).

(SC2034)